dtfs-central-api/src/helpers/calculate-ukef-share-of-utilisation.test.ts

```diff
@@ -0,0 +1,23 @@
+import { calculateUkefShareOfUtilisation } from './calculate-ukef-share-of-utilisation';
+
+describe('calculateUkefShareOfUtilisation', () => {
+  it('should return the ukef share of the utilisation', () => {
+    const utilisation = 100;
+    const coverPercentage = 5;
+
+    const result = calculateUkefShareOfUtilisation(utilisation, coverPercentage);
+
+    expect(result).toEqual(5);
+  });
+
+  it('should return the ukef share of the utilisation when the result is a decimal', () => {
```

Comment:
The test mentions a decimal, but can't see any decimals?

Reply:
changed